File: Chapter4.py
import cv2
import numpy as np

#Shapes and Texts#

img = np.zeros((512,512,3),np.uint8)
#Changing color from Black to Blue#

img[:] = (255,0,0)
#line

cv2.line(img,(0,0),(img.shape[1],img.shape[0]),(255,255,125),3)
cv2.rectangle(img,(0,0),(250,250),(0,255,255), cv2.FILLED)
cv2.circle(img,(250,250),150,(25,250,120),cv2.FILLED)
cv2.putText(img,"Safir",(250,250),cv2.FONT_HERSHEY_COMPLEX,2,(250,250,250),2)
cv2.imshow("Image", img)
cv2.waitKey(0)

File: Chapter4.cpp
#include <opencv2/imgcodecs.hpp>
#include <opencv2/highgui.hpp>
#include <opencv2/imgproc.hpp>
#include <iostream>

using namespace std;
using namespace cv;

int main() {
	Mat img(512, 512, CV_8UC3, Scalar(255, 255, 255));
	circle(img, Point(256, 256), 155, Scalar(0, 69, 255), FILLED);
	
	rectangle(img, Point(130,226),Point(382,286), Scalar(255, 255, 255), FILLED);
	line(img, Point(130, 296), Point(382, 296), Scalar(255, 255, 255), 3);
	putText(img, "Safir's Learning", Point(137, 262), FONT_HERSHEY_PLAIN, 1.75, Scalar(0, 69, 255), 2);



	imshow("Image", img);
	waitKey(0);





	return 0;
}